.changeset/fifty-spoons-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+'@datadog/ui-extensions-sdk': minor
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Can't comment on the actual lines. But since we're making a minor change anyway (and minor bumps in 0.x.x for SemVer are basically breaking changes), should we pull the `DASHBOARD_TIMEFRAME_CHANGE`, and `DASHBOARD_TEMPLATE_VAR_CHANGE` out of `EventType` and put them in some kind of `DeprecatedEventType` or something?

Reply:
Let me think through what would happen here: 

For new apps, this would mean they get typescript errors if they try to use that event, which is nice!

For existing apps on upgrade, this will mean they'll get typescript yelling at them so they will be FORCED to fix the problem. That's kinda nice, i suppose, but typescript won't have a [nice deprecation warning](https://github.com/DataDog/apps/pull/114/files#diff-33921cd9d0560e073ea9422e73bb99aa62be36d7ba833c14da81fb7afa698e4eR55), it will just yell at you but won't tell you how to fix it. That's not as nice of an experience. I'm torn between the forcing function and nice helptext. What do you think?